lib/widgets/quantity_selector.dart

```diff
@@ -97,7 +112,7 @@ class _QuantitySelectorState extends State<QuantitySelector> {
     return GestureDetector(
       onTap: () {
         setState(() {
-          widget.controller.value++;
+          if (widget.controller.value < 10) widget.controller.value++;
```

Comment:
Please add flower braces for better readability.

Reply:
Comparison should be done outside setState. Please try to reduce the computations done within setState as much as possible.